modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// IPI
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+// * installing/ipi-install-post-installation-configuration.adoc
+// UPI
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * post_installation_configuration/post-install-bare-metal-configuration.adoc
+
+ifeval::["{context}" == "ipi-install-post-installation-configuration"]
+:postinstall-bare-metal-ipi:
+endif::[]
+ifeval::["{context}" == "post-install-bare-metal-configuration"]
+:postinstall-bare-metal-upi:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Optional: Creating an additional OVN-Kubernetes `br-ex` bridge network
+
+You can create an additional `br-ex` bridge network that has a different configuration to the default `br-ex` bridge. Creating an additional `br-ex` bridge requires a change to the `configure-ovs.sh` shell script. Changing the shell script requires that you create a machine config manifest file for the network plugin of the Secure Neutron Networking implementation
+
+After you create and edit the manifest file, the Machine Config Operator completes tasks in the following order:
```

Comment:
The below steps should be corrected:

... completes tasks in the following order:

~~~
* Drains nodes one by one in your cluster based on the selected/applied Machine config pool.
* Inject Ignition configuration files into each node in your cluster, so that each node receives the additional `br-ex` bridge network configuration.
* Restarts the node with the newly modified override file in place.
* Configure-ovs.sh is executed referencing the new interface definition.
* `br-ex` and `br-ex1` are brought up on the host node.
* Uncordons the nodes. After all the nodes return to the `Ready` state, the Operator applies the `k8s.ovn.org/l3-gateway-config` annotation to each node.
~~~

Reply:
Thanks. Updated with slight changes.